status-im/status-react
# feature #6250 - added xdai and poa sidechains

fixes #6250 

### Summary:

- Adds two custom RPC networks for POA Network and xDAI Chain
- Adds custom branding for native currencies in those two networks (name, symbol, icon)
- Adds support for fiat prices of POA and xDAI (using POA and DAI prices from cryptocompare, resp.)

### Testing notes (optional):
Note: after this change fiat value of ETHro and ETHri will correctly be calculated as zero.

#### Areas that maybe impacted (optional)
**Functional**

- wallet / transactions
- networks

### Steps to test:
- Switch to new networks and check if native currency is displayed correctly.
- Try sending and receiving POA and xDAI on their respective networks


status: ready
